src/main/java/net/wimpi/modbus/util/SerialParameters.java

```diff
@@ -97,18 +87,16 @@ public SerialParameters(String portName, int baudRate, int flowControlIn, int fl
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this, toStringStyle).append("portName", m_PortName).append("baudRate", m_BaudRate)
-                .append("flowControlIn", getFlowControlInString()).append("flowControlOut", getFlowControlOutString())
-                .append("databits", getDatabitsString()).append("stopbits", getStopbitsString())
-                .append("parity", getParityString()).append("encoding", m_Encoding).append("echo", m_Echo)
-                .append("receiveTimeoutMillis", m_ReceiveTimeoutMillis).toString();
+        return "SerialParameters [m_PortName=" + m_PortName + ", m_BaudRate=" + m_BaudRate + ", m_FlowControlIn="
```

Comment:
Should we prepare for null values? Not perhaps very likely in this specific method

Reply:
Let me think about this..

Reply:
See above..